docs/testing.md

```diff
@@ -43,7 +49,51 @@ Or if you want to run bats from a different repository or branch, edit `centos7-
         bats_forklift_repo: https://github.com/<YOUR_NAME>/forklift.git
         bats_forklift_version: your-branch
 
-## Pipeline Testing
+### To run Bats on an existing VM for development
+
+1. Clone forklift onto your VM (it's recommended to clone into a directory named other than `forklift` so you don't get mixed up):
+```
+git clone git@github.com:theforeman/forklift.git forklift-devel
+```
+2. Clone the Bats repo:
+```
+git clone https://github.com/sstephenson/bats.git
```

Comment:
When using EPEL or Debian, it's preferred to use the package instead. That avoids all the steps below.